### ISSUE TITLE
[14.0] Incomplete TAX statement due to missing partner fiscal position.

### ISSUE THREAD

**Issue by bosd** [OCA/l10n-netherlands#351]
## Tax Statement is missing entries from vendors which do not have a fiscal position set.
The module `l10n_nl_tax_statement` seems to take into account the fiscal postion of the partner.
When the fiscal position on the partner level is not set, the entries from the vendor bills are not included in the tax statment.
From an usability point this is undisered as one does not know that the statement is incomplete.

Likely caused because the fiscal position is not automatically populated on the chart of accounts when one sets the vatnumber on the partner.

## To Reproduce
**Affected versions**: 14.0

1. Install dutch chart of accounts
2. Create a partner by entering name and vat number. (is company)
3. Register an vendor bill for that partner
4.  Generate a tax statement

**Expected behavior**
- Complete generation of tax statement. If there are partners with invoices and no fiscal position raise a warning that the tax statement might be incomplete. Show in the error on which partners that is the case.
- Auto populate the fiscal_position from the vat number (or country code of the partner)

**Additional context**
Could be that this should be handled in odoo natively. But posting here as the effect occurs on the oca tax statement module. And a lot of accounting features have moved to community.

Offcourse an user error is the cause of this as the user should have configured the partner correctly. 
Yet, it is an common error as the fiscal position is not mandatory.
The field is only visible when clicking on the sales&purhase tab. Secondly it is troublesome as one is unaware the statement could be incomplete.

**Comment by bosd** [comment]
@astirpe Your thoughts on this?

**Comment by astirpe** [comment]
@bosd I tried to reproduce the issue you mentioned. I tried both on runbot and locally. In case the fiscal position is not set, I'm always able to get the tax amount under "5b Voorbelasting" (for vendor bills). So to me it seems that the tax statement report works properly, regardless of the fiscal position value.
I also checked the code and I didn't find any possible correlation with the fiscal position.
